### PR TITLE
Revise FeedSourceCard load state to allow deletion for failures

### PR DESCRIPTION
**Pull request by estrattonbailey**
Screenshot 2023-12-11 at 4 04 01 PM
Screenshot 2023-12-11 at 4 04 09 PM

### CODE REVIEW

File: src/view/com/feeds/FeedSourceCard.tsx [hunk 1]
```diff
@@ -105,7 +103,7 @@ export function FeedSourceCardLoaded({
       openModal({
         name: 'confirm',
         title: _(msg`Remove from my feeds`),
-        message: _(msg`Remove ${feed.displayName} from my feeds?`),
+        message: _(msg`Remove ${feed?.displayName} from my feeds?`),
```
Comment (estrattonbailey) [inline src/view/com/feeds/FeedSourceCard.tsx:106]: Just for types

File: src/view/com/feeds/FeedSourceCard.tsx [hunk 2]
```diff
@@ -132,7 +130,70 @@ export function FeedSourceCardLoaded({
...
+        ]}>
+        {showMinimalPlaceholder ? (
+          <FeedLoadingPlaceholder
+            style={{flex: 1}}
+            showTopBorder={false}
+            showLowerPlaceholder={false}
+          />
+        ) : (
+          <FeedLoadingPlaceholder style={{flex: 1}} showTopBorder={false} />
+        )}
```
Comment (estrattonbailey) [inline src/view/com/feeds/FeedSourceCard.tsx:160]: Prev load states are captured here

File: src/view/com/feeds/FeedSourceCard.tsx [hunk 3]
```diff
@@ -132,7 +130,70 @@ export function FeedSourceCardLoaded({
...
+   * This state also captures the scenario where a feed can't load for whatever
+   * reason.
+   */
+  if (!feed || !preferences)
+    return (
+      <View
+        style={[
+          pal.border,
+          {
+            borderTopWidth: showMinimalPlaceholder ? 0 : 1,
```
Comment (estrattonbailey) [inline src/view/com/feeds/FeedSourceCard.tsx:145]: Outlier is `SavedFeeds` usage

**Review by pfrazee — APPROVED** [review]
awesome good job

**Pull request merged**